manifest_staging/charts/csi-secrets-store-provider-azure/templates/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "sscdpa.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "sscdpa.fullname" . }}
```

Comment:
This needs to be `name: {{ template "sscdpa.psp.fullname" . }}`

Reply:
Done